### PR TITLE
Home page how it works section #311

### PR DESCRIPTION
**Pull request by taoliu12**
How it works card section completed.

## Test Plan

repro steps:

1. localhost:3000 
2. Be able to view "how it works section" on homepage

### CODE REVIEW

File: client/src/views/HowItWorks.tsx [hunk 1]
```diff
@@ -1,133 +1,98 @@
...
-
-const nonprofitInstructionList = [
+const nonprofitinstructionsArray = [
   {
-    title: '1. Post your Need.',
-    body: loremIpsum,
-    image: 'https://picsum.photos/seed/nonprofit/600',
+    title: 'Sign up with Givingful',
+    body: 'Create a profile to let our community know about your nonprofit.',
+    image: HowItworks1,
```
Comment (esteban-gs) [inline client/src/views/HowItWorks.tsx:19]: I think we could enhance readability by renaming all these images to what they do, for example: `sign-up-with-givingful.svg`

File: client/src/views/HowItWorksCards.tsx
Comment (esteban-gs) [inline client/src/views/HowItWorksCards.tsx:1]: Both this and the `HowItWorksCard` component can go in the components directory. We are reserving `src/views` for top-level page views.

**Review by esteban-gs — CHANGES_REQUESTED** [review]
@taoliu12 , thanks for working on this. I just have a couple of recommendations.

Also, the build is failing, but not due to your code. I think if we drop the `--legacy-peer-deps` from the line [here](https://github.com/Nonprofit-Exchange-Hub/web-app/blob/318a504a5f4bea6cdeef6ecaca63c6b0e795f443/Dockerfile#L8), it will build successfully.

File: client/src/views/HowItWorksCard.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
+/* eslint-disable prettier/prettier */
```
Comment (esteban-gs) [inline client/src/views/HowItWorksCard.tsx:1]: @taoliu12 , it seems like this eslint comment is causing the build to break.
```suggestion

```

File: client/src/views/HowItWorks.tsx [hunk 3]
```diff
@@ -1,133 +1,98 @@
...
 import { makeStyles } from '@mui/styles';
-import { Box, Grid, Container } from '@mui/material';
+import { Box, Grid } from '@mui/material';
 
 import type { Theme } from '@mui/material/styles';
+import HowItWorksCards from './HowItWorksCards';
+import PeopleSignup from '../assets/HowItWorks/people-signup.svg';
+import PeopleSharing from '../assets/HowItWorks/people-sharing.svg';
+import PeopleWorking from '../assets/HowItWorks/people-working.svg';
+import PeopleFlag from '../assets/HowItWorks/people-flag.svg';
```
Comment (taoliu12) [inline client/src/views/HowItWorks.tsx:11]: these are the new SVG names

**Review by esteban-gs — APPROVED** [review]
Looks good to me!

**Pull request merged**